store/tikv/tests/lock_test.go

```diff
@@ -673,18 +670,16 @@ func (s *testLockSuite) TestBatchResolveTxnFallenBackFromAsyncCommit(c *C) {
 	errMsgMustContain(c, err, "key not exist")
 }
 
-func errMsgMustContain(c *C, err error, msg string) {
-	c.Assert(strings.Contains(err.Error(), msg), IsTrue)
-}
-
-func randKV(keyLen, valLen int) (string, string) {
-	const letters = "abc"
-	k, v := make([]byte, keyLen), make([]byte, valLen)
-	for i := range k {
-		k[i] = letters[rand.Intn(len(letters))]
-	}
-	for i := range v {
-		v[i] = letters[rand.Intn(len(letters))]
+// deduplicateKeys deduplicate the keys, it use sort instead of map to avoid memory allocation.
+func deduplicateKeys(keys [][]byte) [][]byte {
```

Comment:
The duplicate function makes me unhappy to give LGTM

Reply:
sorry I'll find a workaround to make you happy again.

Reply:
test removed.